PR for 12/03/25 telco (#6)
5G-MAG/Tech: pages/Network_APIs/Audio_Streaming/Workflow_for_Audio_Streaming.md

---
layout: default
title: Workflow
parent: Audio Streaming App
grand_parent: Network APIs
nav_order: 0
has_children: false
---

{: .warning }
This documentation is currently **under development and subject to change**. It reflects outcomes elaborated by 5G-MAG members as part of **WI: Network Exposure and APIs Supporting Media Services and Applications**
We welcome and encourage contributions from the broader community. If you are interested in becoming a member of the 5G-MAG and actively participating in shaping this work, please contact the [Project Office](https://www.5g-mag.com/contact)

# Required interactions to exploit network capabilities
  
## Pre-conditions

> * A content provider wishes to stream live segmented-audio over the internet, including mobile networks, to an application running on standards off-the-shelf 3GPP UEs (e.g. smartphones, connected cars, etc.).

Reviewer: ```suggestion
* A application service provider wishes to stream live segmented-audio over the internet, including mobile networks, to an application running on standards off-the-shelf 3GPP UEs (e.g. smartphones, connected cars, etc.).
```

> * The content provider has the ability to use certain **network capabilitues** (e.g. selected from an API catalogue) and has received authentication credentials from the newotk operator authorising their use (when available).

Reviewer: ```suggestion
* The application service provider has the ability to use certain **network capabilities** (e.g. selected from a capability catalogue) and has received authentication credentials from the network operator authorizing their use (when available).
```
